test-utils/runtime/build.rs

```diff
@@ -26,5 +26,13 @@ fn main() {
 		// depend on the stack-size.
 		.append_to_rust_flags("-Clink-arg=-zstack-size=1048576")
 		.import_memory()
-		.build()
+		.build();
+
+	WasmBuilder::new()
+		.with_current_project()
+		.export_heap_base()
+		.import_memory()
+		.set_file_name("wasm_binary_logging_disabled.rs")
+		.enable_feature("disable-logging")
+		.build();
```

Comment:
FIY I tried to debug the test `ensure_runtime_logger_works`, I removed these lines (while I already had the file `wasm_binary_logging_disabled.rs` generated so compilation could work).
And the test finally worked.

I think there is some conflict here somehow.

Reply:
Ahh I did not see that this test failed. I will fix it. :)